src/pages/ReimbursementAccount/BusinessInfo/BusinessInfo.js

```diff
@@ -0,0 +1,152 @@
+import React, {useCallback, useMemo} from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import _ from 'underscore';
+import {parsePhoneNumber} from 'awesome-phonenumber';
+import PropTypes from 'prop-types';
+import useSubStep from '../../../hooks/useSubStep';
+import ONYXKEYS from '../../../ONYXKEYS';
+import {reimbursementAccountPropTypes} from '../reimbursementAccountPropTypes';
+import HeaderWithBackButton from '../../../components/HeaderWithBackButton';
+import InteractiveStepSubHeader from '../../../components/InteractiveStepSubHeader';
+import useLocalize from '../../../hooks/useLocalize';
+import styles from '../../../styles/styles';
+import CONST from '../../../CONST';
+import * as BankAccounts from '../../../libs/actions/BankAccounts';
+import Navigation from '../../../libs/Navigation/Navigation';
+import ROUTES from '../../../ROUTES';
+import ScreenWrapper from '../../../components/ScreenWrapper';
+import getDefaultStateForField from '../utils/getDefaultStateForField';
+import NameBusiness from './substeps/NameBusiness';
+import TaxIdBusiness from './substeps/TaxIdBusiness';
+import WebsiteBusiness from './substeps/WebsiteBusiness';
+import PhoneNumberBusiness from './substeps/PhoneNumberBusiness';
+import AddressBusiness from './substeps/AddressBusiness';
+import TypeBusiness from './substeps/TypeBusiness';
+import IncorporationDateBusiness from './substeps/IncorporationDateBusiness';
+import IncorporationStateBusiness from './substeps/IncorporationStateBusiness';
+import ConfirmationBusiness from './substeps/ConfirmationBusiness';
+import getSubstepValues from '../utils/getSubstepValues';
+import reimbursementAccountDraftPropTypes from '../ReimbursementAccountDraftPropTypes';
+import * as ReimbursementAccountProps from '../reimbursementAccountPropTypes';
+import getInitialSubstepForBusinessInfo from '../utils/getInitialSubstepForBusinessInfo';
+
+const propTypes = {
+    /** Reimbursement account from ONYX */
+    reimbursementAccount: reimbursementAccountPropTypes,
+
+    /** The draft values of the bank account being setup */
+    reimbursementAccountDraft: reimbursementAccountDraftPropTypes,
+
+    /* The workspace policyID */
+    policyID: PropTypes.string,
+};
+
+const defaultProps = {
+    reimbursementAccount: ReimbursementAccountProps.reimbursementAccountDefaultProps,
+    reimbursementAccountDraft: {},
+    policyID: '',
+};
+
+const STEPS_HEADER_HEIGHT = 40;
+// TODO Will most likely come from different place
```

Comment:
do we have a created issue to finish that? Just to make sure that we don't forget :)

Reply:
it's part of another ticket afaik but it's a good point, thanks :+1: